client/state/ui/actions/index.js

```diff
@@ -15,6 +15,7 @@ import 'state/data-layer/wpcom/sites/jitm';
  * Re-exports
  */
 export { default as setRoute } from './set-route';
+export { default as setApplication } from '../application/actions';
```

Comment:
Given that we're setting this on the server side, it feels odd that it lives in the `ui` reducer.

Reply:
I agree at least currently we don't have a need for a way to change apps once its loaded. So we shouldn't have a way to change it. Will remove this.